Options.py

```diff
@@ -23,6 +23,13 @@
     import pathlib
 
 
+def roll_percentage(percentage: typing.Union[int, float]) -> bool:
```

Comment:
Since Generate starts by importing Options, wouldn't it make sense to just fully move roll_percentage here and then change the usage in Generate?

Reply:
Yes 🤦 if you could do that, that'd be great